### ISSUE TITLE
CI failing on Windows (on Julia 1.6 and Julia nightly)

### ISSUE THREAD

**Issue by DilumAluthge** [JuliaEcosystem/PackageAnalyzer.jl#107]
Looks like two CI jobs are failing on `PackageAnalyzer#master`:
1. `CI / Julia 1.6 - windows-latest - x64 (push)`
2. `CI / Julia nightly - windows-latest - x64 (push)`

CI is passing on the remaining jobs.

Is this a known failure?

**Comment by DilumAluthge** [comment]
@ericphanson or @giordano Have you seen these CI failures before?

**Comment by DilumAluthge** [comment]
@ericphanson Any idea what's causing the failure?

**Comment by giordano** [comment]
Maybe we can mark those tests as broken for the time being, at least we restore green CI status.

**Comment by DilumAluthge** [comment]
> Maybe we can mark those tests as broken for the time being, at least we restore green CI status.

Yeah, I can do that. Should I add it to #109, or should I put it in a separate PR?

**Comment by giordano** [comment]
Maybe another PR? #109 is logically separate and looks ready.

**Comment by DilumAluthge** [comment]
SGTM.

**Comment by DilumAluthge** [comment]
> Maybe we can mark those tests as broken for the time being, at least we restore green CI status.

I have implemented this in #111.
